src/common/searchMovies.ts

```diff
@@ -5,12 +5,19 @@ import { DetailedWorkListItem } from "./types/lists";
  * @param works
  * @param searchQuery
  * @returns reviews filtered by searchQuery.
+ * 
  *
  * You can apply filters on the searchQuery with text:value. For example, to filter by title and genre, you can use:
  *
  * "title:jaws genre:horror"
  *
  * Incluidng multiple filters seperated by spaces will implicitly do an AND search between them.
+ * 
+ * Supported filters:
```

Comment:
I personally don't like keeping documentation like this in the code, its too easy for this to get out of date. Do we need it? I think the code is readable enough to understand these are the available types?